lib/blpapi-wrapper.ts

```diff
@@ -69,15 +69,29 @@ var EVENT_TYPE = {
   , PARTIAL_RESPONSE: 'PARTIAL_RESPONSE'
 };
 
+// Mapping of request types to response names to listen for.
+// The strings are taken from section A of the BLPAPI Developer's Guide, and are organized by
+// service.
+var REQUEST_TO_RESPONSE_MAP: { [index: string]: string; } = {
```

Comment:
Part of me wants to advocate for a switch/function for the fall-through for `fieldResponse`.  However, I don't feel that strongly about it.